src/include/executor/execdesc.h

```diff
@@ -245,6 +245,7 @@ typedef struct QueryDesc
 	PlannedStmt *plannedstmt;	/* planner's output, or null if utility */
 	Node	   *utilitystmt;	/* utility statement, or null */
 	const char *sourceText;		/* source text of the query */
+	int		    command_id;		/* id of the query */
```

Comment:
What about placing command_id at the end of the structure? In this case we'll have more binary compatibility: only size of structure will be changed, but fields order will be the same

Reply:
I thought about it... Now the new field is within the section 
```
/* These fields are provided by CreateQueryDesc */
```
If we move it to the end, this logical section will be split. Are you Ok with it?